README.md

```diff
@@ -113,16 +113,27 @@ There are two kinds of users that can access our API: **Project** and **Member**
 - `Member` is the one you use when you log into our webpage with your e-mail.
 - `Project` is designed for integrations and is the one meant for our SDK.
 
-To inform the user to the SDK, pass it as first argument in all functions that interact with the API:
+To inform the user to the SDK, as a keyword argument `user`
 
 ```elixir
-balance = StarkBank.Balance.get!(user)
+balance = StarkBank.Balance.get!(user: user)
+```
+
+Or to configure a project as the default user in the `config/config.exs` file:
+
+```elixir
+config :starkbank,
+  project: [
+    environment: :sandbox,
+    id: "9999999999999999",
+    private_key: private_key_content
+  ]
```

Comment:
Maybe adding`import Config` would be helpful here. It took me a while to figure out it was needed for using the config function.

Reply:
Done